### PR TITLE
Wall Stream Layout Migration #71

### PR DESCRIPTION
**Pull request by yurabakhtin**
Issues: https://github.com/humhub/humhub-modules-polls/issues/71 and https://github.com/humhub/tasks/issues/30

### CODE REVIEW

File: widgets/views/form.php [hunk 1]
```diff
@@ -11,9 +11,16 @@
 
 ?>
 
+<div class="contentForm_options" data-content-component="polls.Poll" style="margin-bottom:10px;margin-top:0">
+    <?= Html::activeTextInput($model,'question', [
+        'placeholder' => Yii::t('PollsModule.widgets_views_pollForm', 'Short question...'),
```
Comment (luke-) [inline widgets/views/form.php:16]: image

image

**Review by luke- — CHANGES_REQUESTED** [review]
Looks very good for me. Just a few little things with the create form.

**Comment by buddh4** [comment]
@yurabakhtin could you change the richtext layout to `LAYOUT_INLINE` according to https://github.com/humhub/humhub-modules-polls/issues/73

**Comment by yurabakhtin** [comment]
@luke- @buddh4 Changes for new layout in commit https://github.com/humhub/humhub-modules-polls/pull/74/commits/5c18c4795c12db01dd0021414bf36dbdd68f2091:

> could you change the richtext layout to LAYOUT_INLINE according to #73

preview_layout

File: widgets/views/form.php (hunk 1 above)
Comment (yurabakhtin) [reply widgets/views/form.php]: done

**Comment by luke-** [comment]
For me it now works as expected.

However, from a UI point of view, I don't like the 4 text inputs placed directly below each other.  
image

Maybe we could:

- Add 1-2 more rows to the description field per Default
- Place a grey "?" in the beginning of each "Answer" text input (similar to the trash can)
- Add headline "Possible answers:" above the Answer section.

What do you think?

**Comment by orangiene** [comment]
hi all,
here are three suggestions for the form fields. I
06_Stream_Polls-input-fields
 think the simplest version is the first one. it's your choice :)

**Comment by luke-** [comment]
@orangiene Thanks for the variants!

I don't want to change too much in this stage and also in comparison to the current design.

@yurabakhtin As first step, can you please try following points:
- Make description multi line
- Add blank line between Title&Description and the answer section
- Maybe: Add title above answers
- Maybe: Make checkboxes two column

**Comment by yurabakhtin** [comment]
@luke- Please check:
new_changes_ui_polls

**Comment by buddh4** [comment]
Looks nice, only the margin between questions/description/answers is a bit to high. Maybe use `ActiveForm::field()` for question and description, which will wrap the inputs in `form-group` divs similar to the edit view.

**Comment by yurabakhtin** [comment]
@buddh4 I tested the `ActiveForm::field()`, the `form-group` has `margin-bottom: 15px;` but we don't need this space when only field "Question" is visible. I.e. on page load when only "Question" is visible we should keep this field with `margin:0`, then on click to the "Question" field we display "Description" below with inline styles:

- first version was `margin:15px 0` for the "Description" field,
- but @luke- wrote "*Add blank line between Title&Description and the answer section*" and I changed that to `margin:30px 0`
- now after your comment I changed to `margin:20px 0 15px` and added same label style for "Answers", please check how it looks:
poll_add_edit

If we will use `ActiveForm::field()` for "Description" then style class `form-group` will adds only `margin-bottom:15px` but we also need `margin-top:~15-30px` because "Question" cannot have `margin-bottom`.

**Comment by buddh4** [comment]
@yurabakhtin So the last screenshot is the current state? I think the margin and labels look fine in this one.

**Comment by yurabakhtin** [comment]
@buddh4 yes, this is current state right after my last changes in commit https://github.com/humhub/humhub-modules-polls/pull/74/commits/a8563b7c8ebfe5e778cf30fdb998d25c1bbd1c91

**Review by luke- — APPROVED** [review]

**Pull request merged**